### DISCUSSION TITLE
GitHub Pages 部署问题

### DISCUSSION THREAD

**Discussion by lateautumn2** [YunYouJun/valaxy#140] [Q&A]
本地pnpm run build 打包出来的文件是正常的,但是push到github上使用gh-pages.xml自动构建,构建出的页面却是文档

**Comment by lateautumn2** [comment]
sorry,已经解决了,是我理解的有一点问题

**Comment by guowei-gong** [comment]
github page 需要在 Settings 中设置一下站点读取的仓库源地址
